neuron_reduce/reducing_methods.py

```diff
@@ -1,24 +1,33 @@
-# This file contains the reduction algorithm itself
-# added the method by Guy to find L and X
-import cmath
+'''
+This file contains the reduction algorithm itself
+added the method by Guy to find L and X
+'''
+import collections
+import contextlib
 import math
-from fractions import Fraction
+import cmath
+
 from neuron import h
-import os
+
+
+CableParams = collections.namedtuple('CableParams',
```

Comment:
make it simpler to keep track of the parameters, instead of putting them in an list sequentially, same for SynapseLocations